src/game-engine/net/outbound-packets.ts

```diff
@@ -296,6 +296,9 @@ export class OutboundPackets {
         const packet = new Packet(110, PacketType.DYNAMIC_LARGE);
         packet.put(widgetId << 16 | childId, 'INT', 'LITTLE_ENDIAN');
         packet.putString(value);
+        if(value === 'gay' || value == 'Book Of Spells') {
```

Comment:
What's this all about? @src52

Reply:
if `value == 'Book of Spells'` is a legit check, use `===` instead of `==` for strict checking.